bin/setup-dev-tools.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/bin/bash
```

Otherwise this gives a `bad substitution` on line 10.